utils/atomic-storybook/.storybook/main.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
     './preset.js',
   ],
   /** @type {import('webpack')['config']['getNormalizedWebpackOptions']} */
-  webpack: async (config) => {
+  webpackFinal: async (config) => {
```

Comment:
That's what the manual says: https://storybook.js.org/docs/react/api/main-config-webpack-final